rgb_camera:
  frame_id: rgb_camera_link
  height: 1200
  width: 1600
  distortion_model: plumb_bob
  D: [0.0007091069098439393, -0.00408537113023305, -0.0003296585047654248, 0.0006370196070704139, 0.055650858265435754]
  K: [1108.9476318359375, 0.0, 795.0526008803499,
      0.0, 1122.2745361328125, 574.2966823486204,
      0.0, 0.0, 1.0]
  R: [1.0, 0.0, 0.0,
      0.0, 1.0, 0.0,
      0.0, 0.0, 1.0]
  P: [1108.9476318359375, 0.0, 795.0526008803499, 0.0,
      0.0, 1122.2745361328125, 574.2966823486204, 0.0,
      0.0, 0.0, 1.0, 0.0]

depth_camera:
  frame_id: depth_camera_link
  height: 480
  width: 640
  distortion_model: plumb_bob
  D: [0.661280353181828, -0.1423830667496564, -0.00016633597211818612, -0.00010782433789381876, 0.11804806839697317]
  K: [414.0901184082031, 0.0, 339.3001909960549,
      0.0, 447.46966552734375, 239.30127972710397,
      0.0, 0.0, 1.0]
  R: [1.0, 0.0, 0.0,
      0.0, 1.0, 0.0,
      0.0, 0.0, 1.0]
  P: [414.0901184082031, 0.0, 339.3001909960549, 0.0,
      0.0, 447.46966552734375, 239.30127972710397, 0.0,
      0.0, 0.0, 1.0, 0.0]

extrinsics:
  depth_to_base_link:
    translation: [0.030594795308249317, 0.00010893223052583068, -0.0022923465032124]  
    rotation_quaternion: [0.9999930795683334, -0.00210655008016876, 0.0028360755041623685, -0.0011661637687825522]  # w,x,y,z
  rgb_to_base_link:
    translation: [0.0, 0.0, 0.0]
    rotation_quaternion: [1.0, 0.0, 0.0, 0.0]